apis/server/network_bridge.go

```diff
@@ -0,0 +1,35 @@
+package server
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/alibaba/pouch/apis/types"
+)
+
+func (s *Server) createNetwork(ctx context.Context, resp http.ResponseWriter, req *http.Request) error {
+	var networkCreateReq types.NetworkCreateRequest
+
+	if err := json.NewDecoder(req.Body).Decode(&networkCreateReq); err != nil {
+		resp.WriteHeader(http.StatusBadRequest)
```

Comment:
Since you will return 400, you should add this status code in swagger.yml, I am afraid.

Reply:
ok